src/hotspot/share/classfile/classLoaderExt.cpp

```diff
@@ -116,8 +117,9 @@ void ClassLoaderExt::process_module_table(JavaThread* current, ModuleEntryTable*
 
 void ClassLoaderExt::setup_module_paths(JavaThread* current) {
   Arguments::assert_is_dumping_archive();
-  _app_module_paths_start_index = ClassLoader::num_boot_classpath_entries() +
-                              ClassLoader::num_app_classpath_entries();
+  int start_index = ClassLoader::num_boot_classpath_entries() +
+                    ClassLoader::num_app_classpath_entries();
+  _app_module_paths_start_index = checked_cast<jshort>(start_index);
```

Comment:
Same question as above

Reply:
Same here.  I didn't want it to not match the declaration.